client/src/component/CohortsTable.js

```diff
@@ -2,32 +2,39 @@ import { Link } from "react-router-dom";
 import { Button } from "./Button";
 
 export const CohortsTable = ({ cohortsData }) => {
-  return (
-    <div>
-      {cohortsData.length > 0 ? (
-        <table>
-          <thead>
-            <tr>
-              <th>Cohort Name</th>
-              <th></th>
-            </tr>
-          </thead>
-          <tbody>
-            {cohortsData.map((cohort) => (
-              <tr key={cohort.id}>
-                <td>{cohort.name}</td>
-                <td>
-                  <Link to={`${cohort.id}`}>
-                    <Button title={"view"} />
-                  </Link>
-                </td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-      ) : (
-        <p>No cohorts found.</p>
-      )}
-    </div>
-  );
-};
\ No newline at end of file
+	return (
+		<div>
+			{cohortsData.length > 0 ? (
+				<table>
+					<thead>
+						<tr>
+							<th>Cohort Name</th>
+							<th>Options</th>
+						</tr>
+					</thead>
+					<tbody>
+						{cohortsData.map((cohort) => (
+							<tr key={cohort.id}>
+								<td>{cohort.name}</td>
+								<td>
+									<span>
+										<Link to={`${cohort.id}`}>
+											<Button title={"view"} />
+										</Link>
+									</span>
```

Comment:
It is not valid HTML to put a button in a link as its navigation you want a link here that can be styled as a button, maybe a ButtonLink component. Also is there any need for the span here?